### ISSUE TITLE
Help investigating double entry for Fabrizio Pandini

### ISSUE THREAD

**Issue by fabriziopandini** [cncf/gitdm.archive#186]
July 2 with https://github.com/cncf/gitdm/pull/160 I have changed my affiliation in [`developers_affiliations2.txt`](https://github.com/cncf/gitdm/blob/acaca98e20a44370901ca08ac5935db39c64bf4c/developers_affiliations2.txt#L30108-L30110)

Then, the 4 of July @lukaszgryglicki with the commit [65d2d95880d8e6b29fb53ea4e25da695d608089c](https://github.com/cncf/gitdm/commit/65d2d95880d8e6b29fb53ea4e25da695d608089c#diff-23e765b09e6062ea307e07ac7652bc34R16436) added another entry for me in [`developers_affiliations1.txt`](https://github.com/cncf/gitdm/blob/master/developers_affiliations1.txt#L17082-L17083), probably using an automatic script

Now, I would like to understand where the second entry comes from, and eventually how to fix my affiliation (nb. I don't recognize fpandini!fpandini-a01.vmware.com as one of my email address, but I would like to understand what is going on before deciding how to proceed...)

Thanks in advance for support 

/assign @lukaszgryglicki

**Comment by lukaszgryglicki** [comment]
This email probably came from some automatic script. I've removed it from our `all_affs.csv` file and regenerated text affiliations files. Its entry was `"fpandini!fpandini-a01.vmware.com","Fabrizio Pandini","VMware","","domain"` which means that affiliation was determined by the "last chance" algorithm "domain". That email must appear in some git logfile from 1000s of repos processed by gitdm.
I've found such commit in thsi repo actually `cncf/gitdm`, see:
```
commit 442fe8cc6e6906e772ed21aebad54729eec50c89
Author: Fabrizio Pandini <fpandini@fpandini-a01.vmware.com>
Date:   Mon Jul 1 14:53:44 2019 +0200

    change affiliation

3 2 developers_affiliations2.txt
```
Which means that someone (maybe even you - I don't know) marked commit 442fe8cc6e6906e772ed21aebad54729eec50c89 as `Fabrizio Pandini <fpandini@fpandini-a01.vmware.com>` and then `cncf/gitdm` identified this email as VMware.

**Comment by lukaszgryglicki** [comment]
Even `git show` can be use to see that this emails was used for commit: `git show 442fe8cc6e6906e772ed21aebad54729eec50c89`:
```
commit 442fe8cc6e6906e772ed21aebad54729eec50c89
Author: Fabrizio Pandini <fpandini@fpandini-a01.vmware.com>
Date:   Mon Jul 1 14:53:44 2019 +0200

    change affiliation

diff --git a/developers_affiliations2.txt b/developers_affiliations2.txt
index 328cdf6b..9ec78ebc 100644
--- a/developers_affiliations2.txt
+++ b/developers_affiliations2.txt
@@ -26930,8 +26930,9 @@ fabianvf: fabianvf!users.noreply.github.com
        Red Hat
 fabien: f-ld!users.noreply.github.com, f.ld!free.fr
        Independent
-fabriziopandini: fabrizio.pandini!gmail.com
-       Unicredit
+fabriziopandini: fpandini!vmware.com, fabrizio.pandini!gmail.com
+       VMware
+       Unicredit until 2019-06-30
 fafhrd91: fafhrd91!gmail.com
        Microsoft
 fahad: fahad!onlab.us
```
I've fixed (removed this affiliation) here: ddec3e1508be8b5b60676c4374cf7e644762c2ba

**Comment by fabriziopandini** [comment]
@lukaszgryglicki thanks for the quick answer, awesome!
